p2p/src/net/libp2p/backend.rs

```diff
@@ -73,6 +73,9 @@ impl Backend {
                     SwarmEvent::NewListenAddr { address, .. } => {
                         log::trace!("new listen address {:?}", address);
                     }
+                    SwarmEvent::BannedPeer { peer_id, .. } => {
```

Comment:
Please avoid using `..`. This is for the same reason why we don't use catch-all `_` in match. You *want* the compiler to complain at every point these structs/arms are used if you change something about them. This will basically silence the compiler.

Reply:
Fixed, although in this case the front-end code doesn't need extra information in order to ban the peer so I don't really see the harm in ignoring rest of the fields.